### ISSUE TITLE
Render file tree nodes as accessible links

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-code-manager#1032]
### Describe the problem and steps to reproduce it:

* Browse a version or view a comparison. Example: https://code.addons-dev.allizom.org/en-US/browse/616959/versions/1688394/?path=popup%2Fquicknote.css
* In this example, open the *popup* folder and click on a file like `quicknote.html`

### What happened?

The `quicknote.html` file loads as expected but the click target is not an anchor link.

### What did you expect to happen?

The click target should be an anchor link. This will solve some accessibility issues. For example, try right-clicking `quicknote.html` to open it in a new tab. You can't do it.

### Anything else we should know?

We rely on [`ListGroup.Item`](https://react-bootstrap.github.io/components/list-group/#list-group-item-props) so I think we can use `as="a"` for anchors but we probably need to add an `href`, too.

**Comment by ruchikabgosain** [comment]
hey @kumar303, i would love to take this up!

**Comment by kumar303** [comment]
Hi @ruchikabgosain. Thanks for your interest! Out of all the issues you commented on, this is probably a good one to start with. 

You'll need to start by going to https://addons-dev.allizom.org/en-US/firefox/ (note the dev domain), register an account, and message me at kmcmillan@mozilla.com with the exact email you used. I suggest using a plus sign to indicate that it will be a *reviewer* account (for reviewing add-ons), something like `kmcmillan+reviewer@mozilla.com`, just as an example. After that I can grant you access to https://code.addons-dev.allizom.org/ which you need also for local development.

**Comment by ruchikabgosain** [comment]
Done :D

**Comment by ruchikabgosain** [comment]
hey @kumar303 could you let me know which files need editing?

**Comment by mirefly** [comment]
@ruchikabgosain  It looks like this line,
https://github.com/mozilla/addons-code-manager/blob/be72debbfb3790028cecbb3bad7b142ea4b0e5d7/src/components/FileTreeNode/index.tsx#L241
and respective tests in[ `FileTreeNode/index.spec.tsc`](https://github.com/mozilla/addons-code-manager/blob/master/src/components/FileTreeNode/index.spec.tsx)

**Comment by kumar303** [comment]
Sorry @ruchikabgosain , I almost missed your question! Thanks, @mirefly -- yes, I think that is the correct place to fix this.
